views/templates/hook/displayBlockProduct.tpl

```diff
@@ -31,7 +31,11 @@
         >
             <span class="item-product">
                 {if $block['icon'] != 'undefined'}
-                    <img class="svg" src="{if $block['icon']}{$block['icon']}{elseif $block['custom_icon']}{$block['custom_icon']}{/if}">
+                    {if $block['icon']}
+                    <img class="svg invisible" src="{$block['icon']}">
```

Comment:
Consider indenting this line and line 37 as done in displayBlocWhite.tpl

Reply:
Done